hw5/files.py

```diff
@@ -0,0 +1,17 @@
+def read_csv(path):
+    data = []
+    with open(path, 'r') as f:
+        for line in f:
+            data.append(line[:-1].split(','))
+    return data[0], data[1:]
+
+
+def write_to_csv(filename, data, columns=None, sep=','):
+    if type(data) == dict:
```

Comment:
if the data is a dict, then its keys can be used as columns. Why do you need a separate `columns` arg?

Reply:
Data key here is store id and value is price mean value